errtracker/errtracker.go

```diff
@@ -91,6 +91,13 @@ func snapConfineProfileDigest(suffix string) string {
 	return fmt.Sprintf("%x", md5.Sum(profileText))
 }
 
+func detectReExec() string {
```

Comment:
The helper is just to match the return type so that it can be used below. I can rename the key in the map if that's what you want